libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.h

```diff
@@ -492,7 +492,13 @@ NS_SWIFT_NAME(UserAccountManager)
 /**
  Switches away from the current user, to a new user context.
  */
-- (void)switchToNewUser NS_SWIFT_NAME(switchToNewUserAccount());
+- (void)switchToNewUser NS_SWIFT_NAME(switchToNewUserAccount()) SFSDK_DEPRECATED(7.2, 8.0, "Use switchToNewUserWithCompletion instead.");
```

Comment:
NB: For Objective-C consumers, it will just be in warning if they use the deprecated API in 7.2, but for Swift consumers, it's an API change.
(We didn't similar things with some of the 7.1 SmartSync changes).